cfgov/scripts/enforcement_transfer.py

```diff
@@ -0,0 +1,165 @@
+)
+from v1.models.learn_page import EnforcementActionPage, EnforcementActionStatus
+from v1.util.migrations import get_stream_data, set_stream_data, strip_tags
+
+
+ENFORCEMENT_PARENT_ID = 1327
+
+STATUS_OPTIONS = [
+    'Post Order/Post Judgment',
+    'Expired/Terminated/Dismissed',
+    'Pending Litigation'
+]
+
+
+def clean_status(s):
+    return s.replace('-', ' ').replace('udge', 'udg').title()
+
+
+def get_metadata_value(item, variable):
+    return strip_tags(item['value']['blob']) or None
+
+
+def transfer():
+    enforcement_actions = BrowseFilterablePage.objects.get(
+        pk=ENFORCEMENT_PARENT_ID
+    )
+
+    errors = []
+    for page in DocumentDetailPage.objects.all():
+        url = page.get_url()
+
+        if not url or 'policy-compliance/enforcement/actions' not in url:
+            continue
+        if 'enforcement-action-definitions' in url:
+            continue
+
+        keys = vars(page)
+
+        old_categories = page.categories.all()
+        if not len(old_categories):
+            errors.append(f'No assigned categories for {url}')
+            continue
+
+        category = CFGOVPageCategory(name=old_categories[0].name)
+
+        tags = page.tags.all()
+        sidefoot = get_stream_data(page, 'sidefoot')
+        sidefoot_fields = []
+        continue_outer_loop = False
+
+        court = ''
+        docket_number = None
+        status = None
+        institution_type = None
+        date_filed = None
+
+        for field in sidefoot:
+            if field['type'] == 'related_metadata':
+                metadata = field['value']['content']
+                for item in metadata:
+                    heading = item['value']['heading']
+
+                    try:
+                        if heading == 'Court':
+                            court = get_metadata_value(item, heading)
+                        elif heading in ['Institution type', 'Institution']:
+                            institution_type = get_metadata_value(
+                                item,
+                                heading
+                            )
+                        elif heading == 'Status':
+                            listed_status = clean_status(
+                                get_metadata_value(item, heading)
+                            )
+                            if listed_status not in STATUS_OPTIONS:
+                                status = STATUS_OPTIONS[0]
+                            else:
+                                status = listed_status
+                        elif heading.lower() in [
+                            'docket number',
+                            'case number',
+                            'case numbers',
+                            'file number',
+                            'file numbers',
+                            'civil action number'
+                        ]:
+                            docket_number = item['value']['blob']
```

Comment:
I actually wanted to preserve the html tags in case there were multiple docket numbers (which would take the form of a `ul` with some `li`s. This is probably hackier than just allowing multiple options though. A change would involve a small update to the model and a migration (probably in a separate PR).

Reply:
Interesting. I didn't realize there were ever multiples. I was just concerned because I saw `<p>` tags in the docket numbers after running the transfer script, and I wasn't sure if that could mess up how those items are displayed. 

But, I just tested transferring a page (page id 1431 is an example) that has a list of docket numbers, and the end result looks fine:
![Screen Shot 2020-03-06 at 5 23 08 PM](https://user-images.githubusercontent.com/4295388/76127231-2b724700-5fcf-11ea-929c-a5f13edd6045.png)

Though the wagtail field looks a little gnarly:
![Screen Shot 2020-03-06 at 5 23 42 PM](https://user-images.githubusercontent.com/4295388/76127267-3e851700-5fcf-11ea-983a-df16ecd11cf7.png)

I'm fine with leaving this code as-is if you want, then we if we want later, we can update the model to officially accommodate multiple docket numbers.

Reply:
Oof, yeah that is a little gnarly. I'll see if I can whip something up so it's more editor-friendly.

Reply:
Completed in https://github.com/cfpb/cfgov-refresh/pull/5560